README.md

```diff
@@ -2,6 +2,46 @@
 
 Meeting notes from the [O3DE Meetings](https://o3de.github.io/sig-core/meetings/).
 
-The [full table of previous meetings](https://o3de.github.io/sig-core/meetings?id=previous-meetings) is available here.
+
 
 # General Resources
+
+# Open 3D Engine Special Interest Group for Core Systems (SIG Core)
+
+This repository holds materials for the Open 3D Engine Foundation Special Interest Group for Core Systems (SIG-Core).
+
+## Description of SIG Core
+We manage the Core Systems needed by O3DE libraries to fulfill the engines requirements. Primarily low-level systems such as memory management, application startup and file input/output are implemented by. See the [SIG's charter](governance/SIG%20Core%20Charter.md) for full details.
+
+## What the SIG does:
+
+* Contributes to changes to O3DE Core Systems in library(AzCore/AzFramework/AzToolsFramework/AzGameFramework).  
```

Comment:
Doesn't the SIG owns most of AzCore/AzFramework? It seems odd just to call out AzGameFramework IMHO

Reply:
Yeah it is quite specific.
I'll remove it from the list of Core Systems